### PR TITLE
Jb49237

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

File: sdk-setup/src/sdk-make-qmltypes [hunk 1]
```diff
@@ -865,6 +873,33 @@ revert_removal_of_manual_additions()
...
 }
 
+find_installed_revision()
+{
+    [[ $1 = self ]] || local -n self=$1
+    local repo=$2
+    local prefix=${3:-v}
+
+    local main_binary=
+    main_binary=$(job_expand_binaries self |awk '{print $1}') || return
```
Comment (vigejolla) [inline sdk-setup/src/sdk-make-qmltypes:883]: Isn't this actually the package name? The word "binary" is usually used for the executables, so this is a bit confusing.

**Review by vigejolla — APPROVED** [review]
LGTM; only one minor nitpick about a variable name

File: sdk-setup/src/sdk-make-qmltypes (hunk 1 above)
Comment (martyone) [reply sdk-setup/src/sdk-make-qmltypes:883]: Yeah it's confusing when you miss the context. In this source the word "binary" always means a "binary (RPM) package". Compare with e.g. man rpmbuild.

Comment (martyone) [reply sdk-setup/src/sdk-make-qmltypes:883]: maybe you missed "main_**binary**=$(job_expand_**binaries** ...)"

Comment (martyone) [reply sdk-setup/src/sdk-make-qmltypes:883]: but that would not likely help you anyway as `job_expand_binaries` may mean many things as well :D

Comment (vigejolla) [reply sdk-setup/src/sdk-make-qmltypes:883]: > Yeah it's confusing when you miss the context. In this source the word "binary" always means a "binary (RPM) package". Compare with e.g. man rpmbuild.

Perhaps main_binary_package would then be better name? Or perhaps just call it "b" and use that for all variables which deal with binary packages? Then you can also say that the reader should be able to understand that in this context "b" always means "binary package".

> maybe you missed "main_**binary**=$(job_expand_**binaries** ...)"

No, I did not. I would have complained about job_expand_binaries as well if it had been part of this PR.

Comment (martyone) [reply sdk-setup/src/sdk-make-qmltypes:883]: Well, I feel it quite balanced WRT to 100 chars wrap margin. One can hardly do anything in the real world not implying some scope. Consistency is more important.

**Pull request merged**